assets/popup.css

```diff
@@ -55,17 +55,21 @@
 
 :root {
     --defaultBackgroundColor: white;
+    --defaultTextColor: black;
 }
 
 @media (prefers-color-scheme: dark) {
     :root {
         --defaultBackgroundColor: black;
+        --defaultTextColor: white;
     }
 }
 
 body {
     margin: 0;
     padding: 0;
+    background-color: var(--backgroundColor, var(--defaultBackgroundColor, white));
```

Comment:
This file replaces the global CSS styles from `@arconnect/components`:

```

const GlobalStyle = createGlobalStyle<{ dark: boolean }>`
  @import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600;700;800&display=swap');

  body {
    background-color: ${(props) => (props.dark ? "#000" : "#fff")};
    color: ${(props) => (props.dark ? "#fff" : "#000")};
    font-family: 'Manrope VF', 'Manrope', sans-serif;
    letter-spacing: .5px;
  }

  input, textarea, button, select {
    font-family: 'Manrope VF', 'Manrope', sans-serif;
    letter-spacing: .5px;
  }
`;

```

Reply:
👍